apps/api/src/domain/models/resource.entity.ts

```diff
@@ -91,14 +91,16 @@ export abstract class Resource extends Aggregate {
         return getAllowedContextsForModel(this.type);
     }
 
-    getContributions(): { contributorId: string; eventType: string; date: number }[] {
-        return this.eventHistory.flatMap(({ meta: { contributorIds, dateCreated }, type }) =>
-            contributorIds.map((contributorId) => ({
-                contributorId,
-                eventType: type,
-                date: dateCreated,
-            }))
-        );
+    getContributions(): {
```

Comment:
Is this temporary? Will it move to the view later?

Reply:
It's not really a problem. The issue is leveraging this to project views of domain models (which requires inefficient joins over event-sourced state). We are "strangnling" this old pattern out. But the method may still be useful for test assertions or otherwise in the context of the domain.